Bundle/GoogleTests.mm

```diff
@@ -172,11 +178,15 @@ @implementation GoogleTestLoader
 + (void)load {
     NSBundle *bundle = [NSBundle bundleForClass:self];
     [[NSNotificationCenter defaultCenter] addObserverForName:NSBundleDidLoadNotification object:bundle queue:nil usingBlock:^(NSNotification *notification) {
-        [self registerTestClasses];
+        [GoogleTestCase initialize];
```

Comment:
Note that this was done because `registerTestClasses` could implicitly invoke `[GoogleTestCase initialize]` and cause duplicated class registration because check `if ([GoogleTestFilterMap count])` will fail.